api/src/main/java/com/velocitypowered/api/command/LegacyCommand.java

```diff
@@ -0,0 +1,8 @@
+package com.velocitypowered.api.command;
+
+/**
+ * A legacy 1.12-style command.
```

Comment:
`A legacy command, modelled after the convention popularized by Bukkit and BungeeCord. Prefer {@code BrigadierCommand} if possible, which is also backwards-compatible with older clients.`

Assuming (of course) that Brigadier support in Velocity does support older clients. Test this perhaps?